src/sangeeta/BankingSystem.java

```diff
@@ -0,0 +1,167 @@
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI
+*/
+
+package sangeeta;
+
+public class BankingSystem {
+	
+	int iBalance = 10000;
+	int sBalance = 15000;
+	int hBalance = 5000;
+	int bBalance = 8000;
+	int iDebitCount = 0, sDebitCount = 0, hDebitCount = 0, bDebitCount = 0;
+	int iCreditCount = 0, sCreditCount = 0, hCreditCount = 0, bCreditCount = 0;
+	static int totalCreditCount, totalDebitCount;
+	
+	 void debitOperation(String bank, int amt) {
+				switch(bank) {
+					case "icici":
+						if(iBalance>=amt) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method